Sources/ScipioKit/DescriptionPackage.swift

```diff
@@ -182,19 +189,27 @@ extension DescriptionPackage {
         return products.reversed()
     }
 
-    private func targetsToBuild() throws -> Set<ResolvedTarget> {
+    private func targetsToBuild() throws -> [ScipioResolvedModule] {
```

Comment:
Since Swift 6, `modules` returns `IdentifiableSet`, it's not available in the old SwiftPM, so it converts it to a regular Array.